event-streams/src/event-streams-v1.0-wd01.md

```diff
@@ -0,0 +1,547 @@
+- `consumer` is the receiver of events from the event log.
+- `event log node` is where AMQP links of senders and receivers attach.
+
+
+## 2.1 Partition support
+
+An event log managed by the event stream node MAY be split into multiple
+parallel partitions that are reachable under the same network address. A single
+event log might also be shared by multiple AMQP containers/nodes that each have
+a separate network address and are each responsible for one or more partitions.
+
+The association between a producer/consumer and a partition MAY be chosen using
+link properties when the link is being attached. The bi-directional negotiation
+capabilities of AMQP apply here: A consumer MAY request being associated with a
+particular partition, but the responding event log node MAY also assign a
+partition if none has been requested.
+
+A producer's `send` link being attached to an event log node MAY request
+association with a specific partition using the [`event-streams-partition` link
+property](#211-event-streams-partition-link-property). If the requested
+partition does not exist, attaching the link MUST be rejected by the event log
+node. The event log node MAY assign a partition when none has been requested. It
+MUST NOT assign a partition other than the one that has been requested.
+
+If no partition is assigned, the default behavior MUST be for a `send` link to
+be partition agnostic and for any partition-related hints to flow via delivery
+annotations, which are defined further below in this section.
+
+A consumer's `receive` link being attached to an event log node MAY request
+association with a specific partition using the the [`event-streams-partition`
+link property](#211-event-streams-partition-link-property). If the requested
+partition does not exist, attaching the link MUST fail. If the event log is
+partitioned and no association has been requested, the event log node MUST
+associate a partition with the link.
+
+For both `send` and `receive` links, the effective associated partition MUST be
+expressed with the [`event-streams-partition` link
+property](#211-event-streams-partition-link-property) set by the event log node.
+
+### 2.1.1 event-streams-partition link property
+
+The `event-streams-partition` link property is a `symbol` value and describes
+the partition with which the link is associated. When set by the producer or
+consumer, the property value is a request for association with the given
+partition. When set by the event log node, the value is the effective
+association.
+
+When not set for a producer 'send' link, the link is partition-agnostic.
+
+### 2.1.2 event-streams-partition delivery annotation
+
+The `event-streams-partition` delivery annotation is a `symbol` value and
+describes the partition to which the message carrying it MUST be routed or from
+which partition is has been retrieved.
+
+If the link over which a message is routed is not partition agnostic with or if
```

Comment:
just to be sure it seems to confirm that logic to determine the destination partition should be on the producer side then?
If the link is partition agnostic then the producer MUST have to set this delivery annotation for specifying the destination partition?